lib/components/Editor/CustomExtensions/constants.js

```diff
@@ -9,7 +9,7 @@ export const DEFAULT_CONFIG = {
   emojiPicker: {},
   customCommands: {},
   characterCount: {},
-  focus: { mode: "shallowest" },
-  starterKit: { document: false, codeBlock: false, code: false },
```

Comment:
Without these changes, there was an error persistent in the useCustomExtensions.js file.